packages/graphback-datasync/src/helpers/createDataSync.ts

```diff
@@ -0,0 +1,23 @@
+import { GraphQLSchema } from 'graphql';
+import { Db } from 'mongodb';
+import { GraphbackAPIConfig, GraphbackAPI, buildGraphbackAPI, GraphbackConfig } from "graphback";
+import { PubSub } from 'graphql-subscriptions';
+import { createDataSyncMongoDbProvider, createDataSyncConflictProviderCreator } from '../providers';
+import { ConflictError, ConflictMetadata, DataSyncModelConflictConfig } from "../util";
+import { createDataSyncCRUDService } from '../services';
+import { DataSyncPlugin } from '../DataSyncPlugin';
+
+type DataSyncGraphbackConfig = Partial<GraphbackConfig>
+
+export function createDataSyncAPI(model: string | GraphQLSchema, db: Db, dataSyncConfigMap: { [modelName: string]: DataSyncModelConflictConfig} = {}, graphbackConfig: DataSyncGraphbackConfig = {}): GraphbackAPI {
```

Comment:
I think 5 arguments is too complicated for the user API, what do you think about reducing it to max 2? 

`DataSyncGraphbackConfig` can copy the required config props (crud, plugins) from [`GraphbackAPIConfig`](https://github.com/aerogear/graphback/blob/ce7236aa84cf053d0e9090115ab5b2b95d3276e4/packages/graphback/src/buildGraphbackAPI.ts#L7). This means very minimal repetition, but greatly simplifies user experience, which is paramount IMO.

```ts
export function createDataSyncAPI(model: string | GraphQLSchema, config: DataSyncGraphbackConfig): GraphbackAPI {
```

Thoughts? Do you see any complications that might arise from this? cc @machi1990